.changeset/fuzzy-falcons-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/identity-auth-frontend': minor
```

Comment:
This will bump the version defined in `package.json` from `0.1.0` to `0.2.0` which might not be what you want as it's the initial release? You can run `pnpm changeset version` to make the version bump and CHANGELOG update locally which you can then tweak as needed and commit.

Reply:
Ah great! Will this still trigger a release to NPM if I do this locally?

Reply:
Yeah, if those are committed it will trigger a release when this PR is merged.